Num-Plate-Detector/number_plate.py

```diff
@@ -0,0 +1,82 @@
+# importing libraries
+import numpy as np
+import cv2
+import  imutils
+import sys
+import pytesseract
+import pandas as pd
+import time
+
+image = cv2.imread('car.jpeg')
+
```

Comment:
@mehabhalodiya take the input path of image from user

Reply:
```
pytesseract.pytesseract.tesseract_cmd = /usr/share/tesseract-ocr/4.00/tessdata
```
Like this?

Reply:
It will get more complicated as it would be different for various OS. Everything will get change!

Reply:
`pytesseract.pytesseract.tesseract_cmd =  input("Enter the path of tesseract in your local system") `
Add this line. This way the user will enter the path where tesseract is present in their local system

Reply:
Also add a readme file

Reply:
> `pytesseract.pytesseract.tesseract_cmd = input("Enter the path of tesseract in your local system") `
> Add this line. This way the user will enter the path where tesseract is present in their local system

Only this one line! Right?

Reply:
@mehabhalodiya instead of pinging again, you can do all your changes, run the script locally, test it by giving inputs and then put the screenshot here

Reply:
I am only getting this!

![npd](https://user-images.githubusercontent.com/73488906/112467451-1492c400-8d8d-11eb-8a64-4c2259555ba3.png)